uber/site_sections/preregistration.py

```diff
@@ -260,7 +260,7 @@ def dealer_confirmation(self, session, id):
     @log_pageview
     def group_members(self, session, id, message=''):
         group = session.group(id)
-        charge = Charge([group, group.leader]) if group.leader else Charge(group)
+        charge = Charge(group)
```

Comment:
Ok, I think we still want to explicitly add `group.leader` as one of the `targets` for `Charge`, because `group.attendees` will not necessarily include the `leader`.

But I am out of my depth when it regarding what is _supposed to happen_. I am not a domain expert here :(